flycheck.el

```diff
@@ -4263,6 +4276,24 @@ universal prefix arg, and only the id with normal prefix arg."
     (`(eval ,_) t)
     (_ nil)))
 
+(defun flycheck-default-directory-wrapper (checker)
+  "Function returning apropriate current working directory.
+
+Work out the working directory from which the CHECKER command
+will be called.  The default is the `default-directory' of the source-buffer
+from which flycheck is calling the checker.
+:default-directory can be a string that contains a fixed directory path or a
+function that will return a path.  The function needs to return an absolute path
+not a relative one."
+  (let* ((cwd (flycheck-checker-get checker 'default-directory))
+         (default-directory (or (and (functionp cwd) (funcall cwd))
+                                (and (stringp cwd) (file-truename cwd))
```

Comment:
I don't think that this feature has any use.  Why would you ever want to give a constant string?

I think we can safely remove this part.